src/codeu/chat/common/Writeable.java

```diff
@@ -0,0 +1,21 @@
+package codeu.chat.common;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import codeu.chat.common.User;
+import codeu.chat.common.Message;
+import codeu.chat.common.ConversationHeader;
+import java.lang.String;
+
+public class Writeable {
```

Comment:
This should probably be an interface instead of a class. That way, you can just leave method signatures below instead of providing a default invalid implementation.